amplify/backend/function/putUser/src/index.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+import gremlin_python
+from gremlin_python.driver import client, serializer
+
+def lambda_handler(event, context):
+    endpoint = os.environ['NEPTUNE_ENDPOINT']
+    port = os.environ['NEPTUNE_PORT']
+
+    # Connect to Neptune
+    db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
+
+    userData = json.loads(event["body"])["userInfo"]
```

Comment:
Do you have to user the `json.loads()` method here? In `postUser/src/index.py` you set `userData = event["userInfo"]`.

Reply:
I believe since the postUser endpoint is at its own endpoint, lambda by default makes the "event" object a dictionary of the body of the request. At the other endpoints however the "event" object has many other parameters like "URI endpoint" and other useful information. This includes "body" which is the body of the request, but in string format.